mlflow/utils/search_utils.py

```diff
@@ -1087,12 +1088,19 @@ def _invalid_statement_token_search_model_registry(cls, token):
 
 
 class SearchModelVersionUtils(SearchUtils):
-    NUMERIC_ATTRIBUTES = {"creation_timestamp", "last_updated_timestamp"}
+    NUMERIC_ATTRIBUTES = {"version_number", "creation_timestamp", "last_updated_timestamp"}
```

Comment:
```suggestion
    NUMERIC_ATTRIBUTES = {"version", "creation_timestamp", "last_updated_timestamp"}
```

Can we change `version_number` to `version`?

Reply:
Actually no, because version is a keyword in sqlparse, I map it to version_number basically the same reason as 'source_path' versus 'source'.

Reply:
Got it. We could implement our own parser to avoid this issue.